backend/src/models/associationSetup.ts

```diff
@@ -47,17 +47,19 @@ export class AssociationSetup {
   // Creates some initial data in database for dev purposes.
```

Comment:
are these necessary? I believe associationSetup is used in production rather than for testing so it should stay clean

Reply:
this is what @JDziewonski98 did to fix the mock data